src/tox/venv.py

```diff
@@ -35,13 +35,13 @@ def writeconfig(self, path):
             lines.append("{} {}".format(*dep))
         path.ensure()
         path.write("\n".join(lines))
+        return lines
```

Comment:
(perhaps I'm not seeing it) is this return value used?

Reply:
indeed not needed 👍